app/src/main/java/org/kiwix/kiwixmobile/language/composables/LoadingIndicator.kt

```diff
@@ -15,13 +15,22 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.language.adapter
 
-import org.kiwix.kiwixmobile.core.base.adapter.AdapterDelegate
-import org.kiwix.kiwixmobile.core.base.adapter.BaseDelegateAdapter
+package org.kiwix.kiwixmobile.language.composables
 
-class LanguageAdapter(
-  vararg delegates: AdapterDelegate<LanguageListItem>
-) : BaseDelegateAdapter<LanguageListItem>(*delegates) {
-  override fun getIdFor(item: LanguageListItem) = item.id
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun LoadingIndicator() {
```

Comment:
@jackq97 I have created a reusable `ContentLoadingProgressBar` in https://github.com/kiwix/kiwix-android/pull/4268/files#diff-e02e2bf321ca88e93589bfabcdc917309398308c5716b7b8d7609efd0ee54407R36 according to our theme, but that is not merged yet. But you can take a reference from there: otherwise I have to change it later.